config.TEMPLATE.inc.php

```diff
@@ -282,7 +282,7 @@
 [email]
 
 ; Default method to send emails
-; Available options: sendmail, smtp, log
+; Available options: sendmail, smtp, log, disabled
```

Comment:
I've added this option mainly to hide error messages from the build log.
We also could use the "log" for it, but then it's needed to check how to setup the log output to another file.